### ISSUE TITLE
For non-example images and example prompts, the generation results fall far short of expectations

### ISSUE THREAD

**Issue by wanganjin** [OmniSVG/OmniSVG#25]
I have reason to suspect that example images and example prompts are part of the training set.

**Comment by CRYPTO-GENIK** [comment]
What gave it away? The fact that there are multiple copies of the exact same samples generated? If you look at the slop they showcased you can tell this project is fake.

**Comment by Tomato1101** [comment]
<img width="1092" height="1018" alt="Image" src="https://github.com/user-attachments/assets/4b1693ad-f333-46fc-bf0a-8b84060670bc" />

赞成。这是我训练模型后的集内测试结果，奇数列是训练集Icon，偶数列是模型推理生成Icon。集外prompt生成效果很差

**Comment by YiyingYang12** [comment]
Thank you for raising this concern and sharing your experimental observations. This is indeed an important issue regarding evaluation validity.

## Regarding Training/Test Set Overlap

You raise a valid point. We acknowledge that splitting a test set directly from the training distribution cannot guarantee that test samples are stylistically distinct from training data, which may compromise the assessment of true model generalization capability.

## Our Solution: MMSVGBench

To address this concern, we have released **MMSVGBench**, a dedicated evaluation benchmark:
👉 https://huggingface.co/datasets/OmniSVG/MMSVGBench

This benchmark is **synthetically generated** and specifically designed to provide fair and rigorous evaluation. Key characteristics include:
- **Zero overlap** with training data distribution
- **Controlled diversity** across different complexity levels
- **Fair comparison** across different methods

We have updated our evaluation results using this benchmark in the latest arXiv version:
👉 https://arxiv.org/html/2504.06263v3

## New Model Releases

We have also released improved versions: **OmniSVG1.1_8B** and **OmniSVG1.1_4B**. We welcome you to evaluate these models and provide further feedback—your input is invaluable for our continued development.

Thank you again for your constructive feedback.
